node/tests/hubController-testConfig.json

```diff
@@ -4,23 +4,23 @@
         "opent2tBlob" : {
             "schema":"opent2t.p.light",
             "translator":"opent2t-translator-com-wink-lightbulb",
-            "controlId":"2120684"
+            "controlId":"2167953"
         }
     }, 
     "setResource": { 
         "opent2tBlob" : {
             "schema":"opent2t.p.light",
             "translator":"opent2t-translator-com-wink-lightbulb",
-            "controlId":"2120684"
+            "controlId":"2167953"
         },
-        "deviceId": "",
+        "deviceId": "F8CFB903-58BB-4753-97E0-72BD7DBC7933",
```

Comment:
>97E0 [](start = 40, length = 4)

Reminder to self to remove this and update readme to populate this in pre-running tests